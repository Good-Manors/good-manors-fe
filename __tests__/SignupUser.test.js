import React from 'react';
import { mount } from 'enzyme';
import { Provider } from 'react-redux';
import { createStore } from 'redux';
import SignupUser from '../src/containers/SignupUser';

describe('Renders a SignupUser container', () => {
  it('Should render a SignupUser', () => {
    const getWrapper = (mockStore = createStore()) => mount(
      <Provider store={mockStore}>
        <SignupUser/>
      </Provider>
    );
    expect(getWrapper).toMatchSnapshot();
  });
});
