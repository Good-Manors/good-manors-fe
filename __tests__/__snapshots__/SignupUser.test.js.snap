// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders a SignupUser container Should render a SignupUser 1`] = `[Function]`;
